src/ZMQ.jl

```diff
@@ -4,7 +4,7 @@ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
 
 module ZMQ
 using Compat
-import Compat.String
+import Compat: String, unsafe_string
```

Comment:
Ah, I didn't realize that ZMQ is extending `unsafe_string`. In that case, the export should still be removed and I suspect the `@deprecate` should be conditional on 0.5 to not get method override and depwarn on 0.4-